zulipterminal/helper.py

```diff
@@ -725,16 +725,17 @@ def suppress_output() -> Iterator[None]:
         os.dup2(stdout, 1)
         os.dup2(stderr, 2)
 
+
 def user_friendly_time(time_stamp: str) -> str:  # message_info has a different format
 
     now = datetime.now()
-    
-    if len(time_stamp) > 22: # for message_info, year is included
+
+    if len(time_stamp) > 22:  # for message_info, year is included
         datetime_object = datetime.strptime(
             time_stamp, "%a %b %d %Y %I:%M:%S %p"
         )  # Fri Apr 29 2022 05:15:12 AM is an example
-    else: # for user_info, year not included
-        if time_stamp[-1] == "M": # AM or PM
+    else:  # for user_info, year not included
```

Comment:
We'll want to keep these lint fixes in the commit in which they were introduced. If you do an interactive rebase and edit the previous commit and run `make fix`, then when you continue the rebase, these changes should likely drop out of this commit (or similarly in later commits, in other complex PRs).

Reply:
Oh, okay! Didn't know, I'll try it out.